api.proto

```diff
@@ -34,6 +34,7 @@ message CreateStreamRequest {
     NullableInt64 autoPauseTime                 = 14; // The amount of time a stream partition can go idle before it is automatically paused
     NullableBool  autoPauseDisableIfSubscribers = 15; // Disables automatic partition pausing when there are subscribers
     NullableInt32 minIsr                        = 16; // The minimum number of replicas that must ack a stream write before it can be committed
+    bool          optimisticConcurrencyControl  = 17; // Set activation mode for Optimistic Concurrency Control on stream. Default is false
```

Comment:
I believe this should follow the `NullableX` pattern still as I envision a corresponding server-side config for this.